src/main/scala/sbtbuildinfo/ScalaCaseClassRenderer.scala

```diff
@@ -17,6 +17,7 @@ case class ScalaCaseClassRenderer(options: Seq[BuildInfoOption], pkg: String, ob
     s"package $pkg",
     "",
     "import scala.Predef._",
+    "import scala.Any",
```

Comment:
Can you explain why this is necessary?

Reply:
When running the scripted test I was having this error `/tmp/sbt_4647108f/caseclassrenderer/target/scala-2.11/src_managed/main/sbt-buildinfo/BuildInfo.scala:8: not found: type Any`
Searching into this I found this may be related bug https://github.com/sbt/sbt/issues/2572